packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -505,6 +505,7 @@ contract PreimageOracle is ISemver {
         if (msg.value < MIN_BOND_SIZE) revert InsufficientBond();
 
         // The caller of `addLeavesLPP` must be an EOA, so that the call inputs are always available in block bodies.
+        // We do NOT use the EOA library here so that we can guarantee that the call inputs are available.
         if (msg.sender != tx.origin) revert NotEOA();
```

Comment:
```suggestion
        // The caller of `initLPP` must be an EOA, so that the call inputs are always available in block bodies.
        // We do NOT use the EOA library here so that we can guarantee that the call inputs are available.
        if (msg.sender != tx.origin) revert NotEOA();
```